lib/src/value_accessors/control_value_accessor.dart

```diff
@@ -91,3 +109,33 @@ abstract class ControlValueAccessor<ModelDataType, ViewDataType> {
     }
   }
 }
+
+class _WrapperValueAccessor<ModelDataType, ViewDataType>
+    extends ControlValueAccessor<ModelDataType, ViewDataType> {
+  final _ModelToViewValueCallback<ModelDataType, ViewDataType>?
+      _modelToViewValue;
+  final _ViewToModelValueCallback<ModelDataType, ViewDataType>?
+      _valueToModelValue;
+
+  _WrapperValueAccessor({
+    _ModelToViewValueCallback<ModelDataType, ViewDataType>? modelToViewValue,
+    _ViewToModelValueCallback<ModelDataType, ViewDataType>? valueToModelValue,
+  })  : _modelToViewValue = modelToViewValue,
+        _valueToModelValue = valueToModelValue;
+
+  @override
+  ViewDataType? modelToViewValue(ModelDataType? modelValue) {
+    if (_modelToViewValue != null && modelValue != null) {
+      return _modelToViewValue!.call(modelValue);
+    }
+    return null;
```

Comment:
You are assuming that the View will automatically receive **null** if the Model data is **null**. 

What if I want to define a default View value different from null (for example the Slider Widget will never receive null as a value, but instead a default **0.0** value as the default value)?

Reply:
```
  @override
  void onControlValueChanged(dynamic value) {
    final effectiveValue = (value == null) ? '' : value.toString();
    _textController.value = _textController.value.copyWith(
      text: effectiveValue,
      selection: TextSelection.collapsed(offset: effectiveValue.length),
      composing: TextRange.empty,
    );

    super.onControlValueChanged(value);
  }
```
I think it will work because the null value is checked here, Or is there something I missed?
Anyway, I will make it required